src/panel/direction/DirectionPanel.jsx

```diff
@@ -310,26 +310,28 @@ export default class DirectionPanel extends React.Component {
     return <DeviceContext.Consumer>
       {isMobile => isMobile
         ? <Fragment>
-          <div className="direction-panel">
+          {!activePreviewRoute && <div className="direction-panel">
             <Flex
               className="direction-panel-header"
               alignItems="center"
               justifyContent="space-between">
               {title}
               <CloseButton onClick={this.onClose} />
             </Flex>
-            {!activePreviewRoute && form}
+            {form}
```

Comment:
are you sure this is needed ? No side effect could occur by still rendering the form ?